OloEngine/src/OloEngine/Renderer/IBLCache.cpp

```
@@ -0,0 +1,578 @@
+        hashCombine(config.UseImportanceSampling ? 1ULL : 0ULL);
+        hashCombine(config.UseSphericalHarmonics ? 1ULL : 0ULL);
+        hashCombine(static_cast<u64>(config.IrradianceResolution));
+        hashCombine(static_cast<u64>(config.PrefilterResolution));
+        hashCombine(static_cast<u64>(config.BRDFLutResolution));
+        hashCombine(static_cast<u64>(config.IrradianceSamples));
+        hashCombine(static_cast<u64>(config.PrefilterSamples));
+
+        return hash;
+    }
+
+    IBLCache::CachePaths IBLCache::GetCachePaths(u64 hash)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)